doc/Makefile

```diff
@@ -72,13 +78,16 @@ UPLOAD_DIR=/srv/docs_scipy_org/doc/numpy-$(RELEASE)
 
 DIST_VARS=SPHINXBUILD="LANG=C PYTHONPATH=$(INSTALL_PPH) python$(PYVER) `which sphinx-build`" PYTHON="PYTHONPATH=$(INSTALL_PPH) python$(PYVER)" 
 
-NUMPYVER:=$(shell $(PYTHON) -c "import numpy; print(numpy.version.git_revision[:10])")
+NUMPYVER:=$(shell $(PYTHON) -c "import numpy; print(numpy.version.git_revision[:10])" 2>/dev/null)
```

Comment:
this is likely right, I just don't know what this does -  why is it needed?

Reply:
previously, if numpy is not importable, an error appeared on the console output and set `NUMPYVER` to `""`. Now it will do so silently.